rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -26,6 +26,4 @@
     name: "{{ item }}"
     state: present
     extra_args: "{{ pip_install_options | default('') }}"
-  with_flattened:
-    - maas_pip_packages
-    - maas_pip_dependencies
+  with_items: maas_pip_packages + maas_pip_dependencies
```

Comment:
Shouldn't this be

```yaml
with_flattened:
  - "{{ maas_pip_packages }}"
  - "{{ maas_pip_dependencies }}"
```

Based on the example [here](https://docs.ansible.com/ansible/playbooks_loops.html#flattening-a-list)